modules/mod/src/main/ModlogApi.scala

```diff
@@ -252,6 +254,15 @@ final class ModlogApi(repo: ModlogRepo, userRepo: UserRepo, slackApi: lila.slack
   def userHistory(userId: User.ID): Fu[List[Modlog]] =
     coll.find($doc("user" -> userId)).sort($sort desc "date").cursor[Modlog]().gather[List](30)
 
+  def countRecentCheatDetected(userId: User.ID): Fu[Int] =
+    coll.countSel(
+      $doc(
+        "user"   -> userId,
+        "action" -> Modlog.cheatDetected,
+        "date" $gte DateTime.now.minusSeconds(3600 * 24 * 30 * 6)
```

Comment:
use `minusDays` or `minusMonths`

Reply:
Will change, I based it on: https://github.com/ornicar/lila/blob/442da0c86a9d54c3cff5645e14d67dfe269a9d0b/modules/mod/src/main/AssessApi.scala#L37 but there's probably a subtility I missed

Reply:
that's just me expecting more from your code than from mine :roll_eyes: